src/overlays/actors/ovl_Bg_Dy_Yoseizo/z_bg_dy_yoseizo.c

```diff
@@ -5,61 +5,54 @@
-void func_80873D14(BgDyYoseizo* this, GlobalContext* globalCtx);
-void func_80873E04(BgDyYoseizo* this, GlobalContext* globalCtx);
-void func_80873EA4(BgDyYoseizo* this, GlobalContext* globalCtx);
-void func_80873FD8(BgDyYoseizo* this, GlobalContext* globalCtx);
-void func_80874304(BgDyYoseizo* this, GlobalContext* globalCtx);
-s32 func_80874B7C(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, BgDyYoseizo* this);
-void func_80874BE0(BgDyYoseizo* this, GlobalContext* globalCtx);
-// func_80874D9C
-void func_80874EAC(BgDyYoseizo* this, GlobalContext* globalCtx);
-void func_808751A0(BgDyYoseizo* this, GlobalContext* globalCtx);
-
-extern UNK_TYPE D_02000130;
-extern UNK_TYPE D_02000160;
-extern UNK_TYPE D_02001020;
-extern UNK_TYPE D_020013E0;
-extern UNK_TYPE D_02001F40;
-extern UNK_TYPE D_020025D0;
-extern UNK_TYPE D_0600092C;
-extern UNK_TYPE D_06001DF0;
-extern UNK_TYPE D_060031C0;
-extern UNK_TYPE D_06004344;
-extern UNK_TYPE D_06005810;
-extern UNK_TYPE D_06005860;
-extern UNK_TYPE D_06005868;
-extern UNK_TYPE D_060058D8;
-extern UNK_TYPE D_06005E60;
-extern UNK_TYPE D_060069E8;
-extern UNK_TYPE D_06007CA8;
-extern UNK_TYPE D_06008698;
-extern UNK_TYPE D_0601C450;
-extern UNK_TYPE D_0601D514;
+void BgDyYoseizo_CheckMagicAcquired(BgDyYoseizo* this, GlobalContext* globalCtx);
+void BgDyYoseizo_ChooseType(BgDyYoseizo* this, GlobalContext* globalCtx);
+void BgDyYoseizo_SetupSpinGrow_NoReward(BgDyYoseizo* this, GlobalContext* globalCtx);
+void BgDyYoseizo_SpinGrow_NoReward(BgDyYoseizo* this, GlobalContext* globalCtx);
+void BgDyYoseizo_Wait_NoReward(BgDyYoseizo* this, GlobalContext* globalCtx);
+void BgDyYoseizo_SetupGreetPlayer_NoReward(BgDyYoseizo* this, GlobalContext* globalCtx);
+void BgDyYoseizo_GreetPlayer_NoReward(BgDyYoseizo* this, GlobalContext* globalCtx);
+void BgDyYoseizo_SetupHealPlayer_NoReward(BgDyYoseizo* this, GlobalContext* globalCtx);
+void BgDyYoseizo_HealPlayer_NoReward(BgDyYoseizo* this, GlobalContext* globalCtx);
+void BgDyYoseizo_SayFarewell_NoReward(BgDyYoseizo* this, GlobalContext* globalCtx);
+void BgDyYoseizo_SetupSpinShrink(BgDyYoseizo* this, GlobalContext* globalCtx);
+void BgDyYoseizo_SpinShrink(BgDyYoseizo* this, GlobalContext* globalCtx);
+void BgDyYoseizo_Vanish(BgDyYoseizo* this, GlobalContext* globalCtx);
+void BgDyYoseizo_SetupSpinGrow_Reward(BgDyYoseizo* this, GlobalContext* globalCtx);
+void BgDyYoseizo_SpinGrowSetupGive_Reward(BgDyYoseizo* this, GlobalContext* globalCtx);
+void BgDyYoseizo_Give_Reward(BgDyYoseizo* this, GlobalContext* globalCtx);
+
+void BgDyYoseizo_ParticleInit(BgDyYoseizo* this, Vec3f* initPos, Vec3f* initVelocity, Vec3f* accel,
+                              Color_RGB8* primColor, Color_RGB8* envColor, f32 scale, s16 life, s16 type);
+void BgDyYoseizo_ParticleUpdate(BgDyYoseizo* this, GlobalContext* globalCtx);
+void BgDyYoseizo_ParticleDraw(BgDyYoseizo* this, GlobalContext* globalCtx);
+
+static s32 unused[] = { 0x0000005D, 0x0000005E, 0x0000005C };
```

Comment:
Although these are unused, they do happen to be the get item ids of Farore's Wind, Nayru's Love, and Din's Fire, respectively.

Reply:
And in yet another ordering